src/bridge/cockpitpipechannel.c

```diff
@@ -66,6 +66,21 @@ typedef struct {
 
 G_DEFINE_TYPE (CockpitPipeChannel, cockpit_pipe_channel, COCKPIT_TYPE_CHANNEL);
 
+GHashTable *internal_fds;
```

Comment:
How does this simple hash table keep track of which fds are for reading and which for writing? Or do we only support receiving read fds for now?